.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -165,11 +167,13 @@ commands:
               fi
               if [ $aws = true ]; then
                 # add AWS url and name
-                nrURL="${nrURL} --url https://${awsApi:-${HLX_AWS_API}}.execute-api.${awsRegion:-${HLX_AWS_REGION}}.amazonaws.com/${actionPackage}/${actionName}/v${actionVersion}/_status_check/healthcheck.json"
-                nrName="${nrName} --name ${nrName}.aws"
+                nrAWSURL="https://${awsApi:-${HLX_AWS_API}}.execute-api.${awsRegion:-${HLX_AWS_REGION}}.amazonaws.com/${actionPackage}/${actionName}/v${actionVersion}/_status_check/healthcheck.json"
+                nrAWSName="${nrName}.aws"
```

Comment:
Do you have a test that uses either `$awsApi` or the default `$HLX_AWS_API`?

Reply:
Might be worthwhile enhancing the test script to have an `environment` map property, whose values are then passed to the script execution...

Reply:
`$awsAPI` yes, `$HLX_AWS_API` not yet...